### DISCUSSION TITLE
Custom menu active class

### DISCUSSION THREAD

**Discussion by patel-hetas** [saadeghi/daisyui#2806] [Q&A] [answered]
Hello guys,
Currently I am working on my portfolio using DaisyUI(great library by the way). I was using the menu component and was trying to customise the active class for a menu item but was not able to find a way around. It would be great if anyone can provide any suggestions on this.

Thank you

**Comment by himanshujangid03** [comment]
Hi,
I hope to get your question correctly. I was facing same error so here's the solution for this.

In Index.css file paste the below code and customize styles according to your preference.

```
/* apply active elements of menu class  */

.active {
  @apply !bg-indigo-950 !text-indigo-50;
}
```

**Reply by NikarashiHatsu** [reply, marked as answer]
That's a good workaround, but, just in case you just want to re-map the class, you could just do this instead:

```css
.active-custom {
  @apply active;
}
```
